source/faq.rst

```diff
@@ -0,0 +1,101 @@
+.. toctree::
+
+FAQ
+===
+
+
+Why joining the EGI Cloud?
+--------------------------
+
+* To support international communities supported by EGI (e.g. `these research
+  communities and applications <https://www.egi.eu/use-cases/>`_ or `these
+  research infrastructures in EOSC-hub <https://eosc-hub.eu/research-communities>`_
+  or `these business pilots in the EOSC Digital Innovation Hub <https://eosc-hub.eu/digital-innovation-hub>`_.
+
+* To participate in e-Infrastructure projects (H2020, EOSC) as an EGI compliant IaaS cloud provider.
+
+* To participate in resource allocation and in pay-for-use campaigns run by EGI.
+
+* To align access policies and operational model of your cloud with
+  international good practices.
+
+* To adopt best practices of multi-cloud federation for the benefit of your
+  local users.
+
+
+Do I lose control on who can access my resources if I join federated cloud?
+---------------------------------------------------------------------------
+
+**No**
+
+EGI uses the concept of Virtual Organisation (VO) to group users. The resource
```

Comment:
I think it would be nice to have a clear/simple page explaining how we use VO, we could refer to it from various places, maybe something to record as an issue somewhere?

Reply:
ok, #46 opened. It would be good to have the common docs in place for this.